tools/xdp/s2n-quic-xdp/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "s2n-quic-xdp"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
Not a huge deal but this is the first release of this crate so we can keep it `0.1.0`. Then again, the versions are arbitrary.

Reply:
Oh true, I didn't even realize this was the first release. I'll keep it 0.1.0, it's easy.